src/index.html

```diff
@@ -1,13 +1,61 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="../src/styles/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <article class="card" data-qa="card">
+      <img 
+        class="card__image"
+        src="./images/imac.jpeg" 
+        alt="APPLE-A1419-iMac-27-Retina-5K-Monoblock-(MNED2UA/A)"
+      >
+
+      <h2 class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <span class="card__product-code">
+        Product code: 195434
+      </span>
+
+      <div class="card__reviews">
+        <ul class="stars stars--4">
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li> 
+          <li class="stars__star"></li>
+        </ul>
+
+        <span class="card__reviews-amount">
+          Reviews: 5
+        </span>
+      </div>
+
+      <p class="card__price">
+        <span class="card__price-text">Price:</span>
+
+        <span class="card__price-amount">$2,199</span>
+      </p>
+
+      <a 
+        href="#"
```

Comment:
The 'alt' attribute should only be used with ```img``` elements, not with ```a``` elements. Remove the 'alt' attribute.